analyses/project-updates/01-generate-project-report.Rmd

```diff
@@ -396,6 +401,9 @@ For more information, please see: `r results_filepath`/`r cohort_value`/analyses
 
 For more information, please see: `r results_filepath`/`r cohort_value`/analyses/cell-types-annotation/plots
 
+   - We used the **`r method_annotation_module`** method(s) for identifying cell types.
+   - We used the **`r annotation_file1`** and **`r annotation_file2`** file(s) for identifying cell types.
```

Comment:
I think it could help clarity to make printing these file names optional depending on the methods used, as it might be a little confusing to print the blank file extension in cases where the `gene_markers` or `reference` annotations weren't used.